src/common/snippets/src/lowered/loop_manager.cpp

```diff
@@ -37,6 +39,19 @@ std::shared_ptr<LoopPort> LoopPort::clone_with_new_expr(const ExpressionPtr& new
     return new_loop_port;
 }
 
+LinearIR::LoopManager::LoopInfo::LoopInfo(size_t work_amount,
+                                          size_t increment,
+                                          const std::vector<LoopPort>& entries,
+                                          const std::vector<LoopPort>& exits,
+                                          bool outer_splited_loop)
+    : m_work_amount(work_amount),
+      m_increment(increment),
+      m_entry_points(entries),
+      m_exit_points(exits),
+      m_outer_splited_loop(outer_splited_loop) {
+    handlers.resize(3);
```

Comment:
Could you tell me please what is the magic number `3`? Is it `First, Main, Last`? Could you please leave the small comment here? Thanks!

Reply:
Done, please check

Reply:
Not actual anymore since handlers now is a separate class